.vscode/launch.json

```diff
@@ -0,0 +1,25 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Jest",
+      "type": "node",
+      "request": "launch",
+      "program": "${workspaceFolder}/scripts/jest.js",
```

Comment:
Can just use `${workspaceFolder}/node_modules/.bin/jest` and remove the script.

Reply:
@yyx990803 `${workspaceFolder}/node_modules/.bin/jest` is not work for all operating system, 
 see this issue: https://github.com/microsoft/vscode-recipes/issues/107.  I encouter this problem several years ago and use this simple script for all my project.....

I just test in my window, it is still not work in latest vscode .....

Reply:
Ok, in this case can you add the reason as comments to the jest script?

Reply:
ok